mmv1/templates/terraform/custom_import/kms_crypto_key_version.go.erb

```diff
@@ -0,0 +1,17 @@
+
+	config := meta.(*Config)
```

Comment:
please standardize indentation on tabs or spaces for readability

Reply:
Will do